.rspec

```diff
@@ -1,2 +1,3 @@
 --format documentation
 --color
+--require spec_helper
```

Comment:
putting this here has bitten me tons of times, but probably someone else told you that it is awesome :trollface: